openvisualizer/client/plugins/msf.py

```diff
@@ -17,7 +15,7 @@ def __init__(self, proxy, mote_id, refresh_rate):
 
     def render(self, ms=None):
         super(MSF, self).render()
-        msf_values = json.loads(ms[MoteState.ST_MSF])
+        # msf_values = json.loads(ms[MoteState.ST_MSF])
```

Comment:
Is this intended to be removed?

Reply:
Yes for now it is. The MSF viewer is for a future feature that is not yet implemented.